app/src/main/res/layout/activity_main.xml

```diff
@@ -7,13 +7,179 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <View
+        android:id="@+id/id_my_profile_up"
+        android:layout_width="@dimen/layout_param"
+        android:layout_height="@dimen/layout_param"
+        android:background="@color/profile_background"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/id_my_profile_down" />
+
+    <View
+        android:id="@+id/id_my_profile_down"
+        android:layout_width="@dimen/layout_param"
+        android:layout_height="@dimen/layout_param"
+        android:background="@color/white"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/id_my_profile_up" />
+
     <TextView
+        android:id="@+id/id_text_view_settings"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginStart="@dimen/margin_header_content"
+        android:layout_marginTop="@dimen/margin_header_content"
+        android:text="@string/text_settings"
+
+        android:textColor="@color/setting_text"
+        android:textSize="@dimen/text_size_settings"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/id_btn_view_logout"
```

Comment:
there is no need to have the "id_" part of the ids names, you could omit this part

Reply:
I have fixed it